SPEC/ovirt-openvswitch.spec

```diff
@@ -0,0 +1,153 @@
+%define ovs_version 2.11
+%define ovs_version_to_replace 2.11.1
+
+%global py_openvswitch python3-openvswitch
+
+Name:           ovirt-openvswitch
+Version:        %{ovs_version}
+Release:        0.2020060501%{?dist}
+Summary:        Wrapper RPM for upgrading OVS to newer versions
+
+Group:          System Environment/Daemons
+License:        Public Domain
+URL:            http://www.openvswitch.org
+BuildArch:      noarch
+
+Requires:       openvswitch%{ovs_version}
+Provides:       openvswitch = %{ovs_version}
+Obsoletes:      openvswitch <= %{ovs_version_to_replace}
```

Comment:
I'm again puzzled by this; how can you obsolete version 2.11.1 and provide version 2.11 ? To my eyes, you're obsoleting the version you're providing.

Did you ever understood how does this work ?

Reply:
You are right, this is odd.
Do you prefer adding epoch or any alternative idea?